packages/cli/src/cli/overview/action.create.overview.ts

```diff
@@ -0,0 +1,176 @@
+    const builder = new CogBuilder(tileMatrix, MaxConcurrencyDefault, logger);
+    const metadata = await builder.build(tiffSource, cutline);
+
+    logger.info({ source, path }, 'CreateOverview: prepareTiles');
+    this.prepareTiles(metadata.files, maxZoom);
+    if (this.tiles.size < 1) throw new Error('Failed to prepare overviews.');
+
+    logger.info({ source, path }, 'CreateOverview: GenerateTiles');
+    await this.generateTiles(path, tileMatrix);
+
+    logger.info({ source, path }, 'CreateOverview: CreatingTarFile');
+    await this.createTar(path, logger);
+
+    logger.info({ source, path }, 'CreateOverview: Finished');
+    return;
+  }
+
+  async prepareTiles(files: NamedBounds[], maxZoom: number): Promise<void> {
+    for (const file of files) {
+      const name = file.name;
+      const [z, x, y] = path.basename(name).replace('.tiff', '').split('-').map(Number);
+      let qk = QuadKey.fromTile({ x, y, z });
+      this.addChildren(qk, maxZoom);
+      while (qk.length > 0) {
+        if (this.tiles.has(qk)) break;
+        this.tiles.add(qk);
+        qk = QuadKey.parent(qk);
+      }
+    }
+  }
+
+  addChildren(qk: string, maxZoom: number): void {
+    if (qk.length >= maxZoom) return;
+    for (const child of QuadKey.children(qk)) {
+      this.tiles.add(child);
+      if (child.length < maxZoom) this.addChildren(child, maxZoom);
+    }
+  }
+
+  async prepareSourceFiles(sources: ChunkSource[]): Promise<TileMatrixSet> {
+    let tileMatrix;
+    for (const source of sources) {
+      const tiff = await CogTiff.create(source);
+      await tiff.getImage(0).loadGeoTiffTags();
+      if (tileMatrix == null) tileMatrix = TileMatrixSets.tryGet(tiff.getImage(0).epsg);
+      const imgBounds = Bounds.fromBbox(tiff.getImage(0).bbox);
+      this.files.push({
+        name: tiff.source.uri,
+        ...imgBounds,
+      });
+    }
+    if (tileMatrix == null) throw new Error('Unable to find the imagery tileMatrix');
+    return tileMatrix;
+  }
+
+  async generateTiles(path: string, tileMatrix: TileMatrixSet): Promise<void> {
+    const promises = [];
+    let currentTiles = Array.from(this.tiles);
+    while (currentTiles.length > 0) {
+      const todo = currentTiles.slice(0, WorkerTaskSize);
+      currentTiles = currentTiles.slice(WorkerTaskSize);
+      const jobTiles: JobTiles = { path, files: this.files, tileMatrix: tileMatrix.identifier, tiles: todo };
+      promises.push(pool.run('tile', jobTiles));
+    }
+
+    await Promise.all(promises);
+    await pool.close();
+  }
+
+  async createTar(path: string, logger: LogType): Promise<void> {
+    const tarFile = 'overview.co.tar';
+    const tarIndex = 'overview.tar.index';
+    const tarFilePath = fsa.join(path, tarFile);
+    const tarIndexPath = fsa.join(path, tarIndex);
+
+    // Create tar file
+    const tiles = await fsa.toArray(fsa.list(`${path}/tiles/`));
+    const cwd = fsa.join(process.cwd(), path);
+    const paths = tiles.map((c) => c.replace(`${cwd}/`, ''));
```

Comment:
you can just strip of `${path}`  we should avoid `process.cwd()`

Reply:
I need to pass `cwd/${path}` into `tar.cwd` which will make sure the no additional path are compressed into tar file.

Reply:
Im not sure I follow,  can you just use the exact same tar creation method as cotar ? 

https://github.com/linz/cotar/blob/master/packages/cli/src/commands/tar.ts#L41

Using `process.cwd()` for anything,  can be super unreliable